blog/Webots-2018-b-release.md

```diff
@@ -0,0 +1,125 @@
+We will be rolling it out across our objects and simulations as time goes on, and this will likely change the appearance of your simulation worlds.
+
+We plan to continue developing advanced graphics features for Webots in the coming months, to complement this rollout.
+
+---
+
+## New Look and Feel
+
+### New Themes
+
+%figure "Webots Night, The new default theme for Webots"
+![dark theme](images/webots_night.png)
+%end
+
+We've done lots of work on user experience with this latest update. 
+Shown above is the new default look for Webots R2018b, Webots Night.
+This new look features clean lines, flat design, and a subdued color palette with highlights in just the right places to make interacting with Webots even more intuitive.
+If you prefer the old look, we've also kept the previous style, in a theme called "Webots Classic".
+There's also a more vibrant dark theme named "Webots Dusk".
+Additionally, the splash screen has been re-designed to match our new themes.
+
+### It's All about Context
+
+We've wanted to streamline actions normally undertaken during normal simulation design workflow. 
+These include editing a robot's controller, seeing a robot's window, deleting an object from the scene, etc.
+So, to improve this, we've added a context menu to the 3D view and Scene Tree!
+
+%figure "A Typical Context Menu for a Selected Robot Node"
+![context menu](images/context_menu.png)
+%end
+
+The menu adapts to the currently selected tree item or object in the 3D view, providing selection-specific actions and options.
+
+Boot up Webots R2018b and have an explore of these new GUI features!
+
+---
+
+## Simulation Reset
+
+Reloading an entire world just to re-run your simulation can be slow, especially for large worlds.
+With this in mind, we have developed an alternative method of resetting a simulation, and our terminology has changed as such.
+"Revert" has become "Reload" and now users can benefit from the new "Reset Simulation" action, also accessible via the [Supervisor API](../reference/supervisor.md#wb_supervisor_simulation_reset).
+
+%figure "Reset Simulation Action"
+![reset simulation](images/reset.png)
+%end
+
+This feature makes resetting even the most complex simulations instantaneous.
+
+---
+
+## New PROTOs
+
+For more realistic and immersive vehicle simulations, we have expanded our available building models and street furniture extensively, with more than 80 new models.
+Below is a preview of just some of these new assets.
+
+%figure "New PROTO models of Buildings and Street Furniture"
+![new objects](images/new_objects.png)
+%end
+
+---
+
+## Extra Goodies
+
+Webots R2018b supports Ubuntu 18.04 LTS out of the box.
+
+The [Speaker API](../reference/speaker.md#speaker-functions) can now load and play all major audio formats.
+
+If you don't want to run a controller for your Robot node, you can now simply use "" as the controller name, to avoid running the void controller unnecessarily.
```

Comment:
I would say "simply use an empty string ("") as the controller name" to be more explicit.